docker/referentielijsten/fixtures/referentielijsten_fixtures.json

```diff
@@ -0,0 +1,39 @@
+[
+{
+    "model": "api.tabel",
```

Comment:
Could you also add a superuser to this fixture? That makes it a bit easier to update/extend this fixture later on